### ISSUE TITLE
Rambo vomits when there is no method description

### ISSUE THREAD

**Issue by danascheider** [danascheider/rambo#22]
Currently, under each method listed for a given route, Rambo relies on there being a method description to generate the case description. When there is no method description given, the following error appears: 
```
    ____   ___     __  ___ ____   ____
   / __ \ /   |   /  |/  // __ ) / __ \
  / /_/ // /| |  / /|_/ // __  |/ / / /
 / _, _// ___ | / /  / // /_/ // /_/ /
/_/ |_|/_/  |_|/_/  /_//_____/ \____/

Generating contract tests...
Error: undefined method `downcase' for nil:NilClass
```
The source of this is `/lib/rspec/templates/example_group_template.erb:9`, where `downcase` is called on `method.description`.

**Comment by danascheider** [comment]
The PR resolving this issue is not being merged yet because of #20 (JSON Test Data 0.7 breaks tests).

**Comment by danascheider** [comment]
Resolved
